app/src/main/res/layout/activity_result_detail.xml

```diff
@@ -52,7 +53,7 @@
 		android:id="@+id/recyclerView"
 		android:layout_width="match_parent"
 		android:layout_height="match_parent"
+		android:clipToPadding="false"
+		android:paddingBottom="66dp"
 		app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-</androidx.coordinatorlayout.widget.CoordinatorLayout>
-
-	<!-- app:layout_scrollFlags="scroll|exitUntilCollapsed|snap" -->
\ No newline at end of file
+</androidx.coordinatorlayout.widget.CoordinatorLayout>    <!-- app:layout_scrollFlags="scroll|exitUntilCollapsed|snap" -->
```

Comment:
why there is a commented line here?

Reply:
is only a placeholder for removed transition